packages/valory/skills/abstract_round_abci/test_tools/integration.py

```diff
@@ -40,46 +36,15 @@
 from aea.multiplexer import Multiplexer
 from aea.protocols.base import Address, Message
 from aea.skills.base import Handler
-from aea_ledger_ethereum import EthereumApi
 from web3 import HTTPProvider, Web3
 from web3.providers import BaseProvider
-from web3.types import Nonce, Wei
-
-from autonomy.test_tools.fixture_helpers import HardHatAMMBaseTest
-from autonomy.test_tools.helpers.contracts import get_register_contract
-
-from packages.open_aea.protocols.signing import SigningMessage
-from packages.valory.protocols.contract_api import ContractApiMessage
-from packages.valory.protocols.contract_api.custom_types import RawTransaction, State
-from packages.valory.protocols.ledger_api import LedgerApiMessage
-from packages.valory.protocols.ledger_api.custom_types import (
-    SignedTransaction,
-    TransactionDigest,
-    TransactionReceipt,
-)
+
+from packages.valory.skills.abstract_round_abci.base import BaseSynchronizedData
 from packages.valory.skills.abstract_round_abci.behaviour_utils import BaseBehaviour
 from packages.valory.skills.abstract_round_abci.test_tools.base import (
     FSMBehaviourBaseCase,
 )
-from packages.valory.skills.liquidity_rebalancing_abci.rounds import (
```

Comment:
The abstract_round_abci is not allowed to depend on packages inheriting from it.